Zc-specification/Zc.adoc

```diff
@@ -284,6 +284,27 @@ The Zcmt extension depends on the <<Zca>> and Zicsr extensions.
 
 |===
 
+[#Zc_formats]
+=== Zc instruction formats
+
+[%header,cols="2,3,2,1,1,1,1,1,1,1,1,1,1"]
+|=====================================================================
+| Format | instructions       | 15 10  | 9  | 8 | 7   | 6 | 5 | 4   | 3 | 2  | 1 | 0

```

Comment:
Looks good to me. The only comment I have is maybe we add a hyphen here or .. to indicate a range ?